docs/e2e_flows.md

```diff
@@ -0,0 +1,22 @@
+# E2E Flows
+
+## Data sent from application to CKAN through Data API 
+In order to send data from your application to CKAN through the Data API you need to follow the next steps:
+
+### Pre-requisities:
+- Ask for a service to the platform administrator.
```

Comment:
A lot of "Ask for" items... Maybe it would bettter to "factorize", having an "Ask for" item and sub-item for each one of the elements.